Description: Replace ancient process.binding('evals') with require('vm')
Author: Jonas Smedegaard <dr@jones.dk>
Copyright: GPL-3+
Last-Update: 2018-01-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test-async.js
+++ b/test/test-async.js
@@ -1806,7 +1806,7 @@
             if(err) return test.done();
 
             // Script -> NodeScript in node v0.6.x
-            var Script = process.binding('evals').Script || process.binding('evals').NodeScript;
+            var Script = require('vm').Script || require('vm').NodeScript;
 
             var s = new Script(content, filename);
             var s2 = new Script(
